teamsupport/models.py

```diff
@@ -1,19 +1,31 @@
 from property_caching import cached_property
 from querylist import QueryList
 
+from teamsupport.services import TeamSupportService
 from teamsupport.errors import MissingArgumentError
 
 
+TICKET_STATUS_NEW = '212203'
+TICKET_TYPE_SUPPORT = '35731'
```

Comment:
> I can't believe they can change.

I'm suggesting that they are different for different TeamSupport customers. I.e. NEW for yola is a different ID than new for some other TeamSupport user.

> Regarding getting them dynamically - how to find them?

I'd assume a Status / Type enumeration API. And then matching (probably exactly) by title, yes.

I don't really see any other options, beyond ignoring this until someone else tries to use this library, and finds that it has hard-coded constants that are Yola-specific.